modules/io/src/writers/xviz-binary-writer.js

```diff
@@ -30,7 +51,12 @@ export function encodeBinaryXVIZ(xvizJson, options) {
   const packedData = packBinaryJson(xvizJson, gltfBuilder, null, options);
 
   // As permitted by glTF, we put all XVIZ data in a top-level subfield.
-  gltfBuilder.addApplicationData('xviz', packedData, {nopack: true});
+  const {compatible} = options;
```

Comment:
can we add a more specific option or name for `compatible`.  This makes me ask compatible to what.

Reply:
We decided to go with `useAVSXVIZExtension` since this is a hidden/undocumented feature we are trying out right now.